tests/deprecated_api/test_remove_1-8.py

```diff
@@ -256,3 +269,34 @@ def test_v1_8_0_deprecated_training_type_plugin_property():
     trainer = Trainer()
     with pytest.deprecated_call(match="in v1.6 and will be removed in v1.8"):
         trainer.training_type_plugin
+
+
+@pytest.mark.parametrize(
+    "cls",
+    [
+        DDPPlugin,
+        DDP2Plugin,
+        DDPSpawnPlugin,
+        pytest.param(DeepSpeedPlugin, marks=RunIf(deepspeed=True)),
+        DataParallelPlugin,
+        DDPFullyShardedPlugin,
+        pytest.param(IPUPlugin, marks=RunIf(ipu=True)),
```

Comment:
This won't run at the moment because we only run ipu tests on the ipu test file

Reply:
Do you think #11170 will land? I can drop the test case completely, move it to the ipu test file, or keep as is

Reply:
I don't know. I would move the test to `test_ipu.py` and add a comment here referencing it.